src/examples/bad_code.cpp

```diff
@@ -0,0 +1,49 @@
+#include "ros/ros.h"
+#include "std_msgs/Float64.h"
+
+ros::Publisher pub;
+//pretend I commented everything
+double calc_pi    (void)
+{
+    ROS_INFO("calculating pi...");
+    double result=0;
+    for (int i=0; i < 100000000; ++i)
```

Comment:
EXTREMELY inefficient calculation of PI. Usually don't need this kind of accuracy...